src/Enzo/enzo_EnzoConfig.cpp

```diff
@@ -168,6 +168,7 @@ EnzoConfig::EnzoConfig() throw ()
 EnzoConfig::~EnzoConfig() throw ()
 {
 #ifdef CONFIG_USE_GRACKLE
+  free(method_grackle_chemistry->grackle_data_file);
```

Comment:
Why free() and not delete? This string was allocated with new.

Reply:
It's because the grackle_data_file is allocated by `strdup` which I believe uses `malloc` internally. I'll double-check this and add a comment documenting this.